### ISSUE TITLE
Strange message was shown when I use i18next cli

### ISSUE THREAD

**Issue by tanosugi** [i18next/i18next-parser#503]
## 🐛 Bug Report

Strange message was shown when I use i18next cli

## To Reproduce

```js
// CLI command I input
i18next
```

## Output
```js
// strange message was shown
LIBERTY LIBERTY LIBERTY
LIBERTY LIBERTY LIBERTY
LIBERTY LIBERTY LIBERTY
                                   !             H|H|H|H|H           H__________________________________             H|§|§|§|H           H|* * * * * *|---------------------|             H|§|∞|§|H           H| * * * * * |---------------------|             H|§|§|§|H           H|* * * * * *|---------------------|             H|H|H|H|H           H| * * * * * |---------------------|             H|H|H|H|H           H|---------------------------------|          ===============        H|---------------------------------|            /| _   _ |          H|---------------------------------|            (| O   O |)          H|---------------------------------|            /|   U   |          H-----------------------------------             |  =/  |           H              _..._/            H              _|I/|_            H      _______/| H |/_______    H     /           / /          H    |          | | /         |  H    |          ||o||          |  H    |    |     

```
## Your Environment

- *runtime version*: i.e. node v17
- *i18next-parser version*: i.e. 5.3.0
- *os*: Linux

**Comment by heymartinadams** [comment]
I can confirm the same happening to me. Also if I install a previous `4.0.1` version.

**Comment by gelaechter** [comment]
This is because i18next-parser depends on the latest version of colors.js
https://github.com/i18next/i18next-parser/blob/054e29a7b0cae3860857cb0cd5b0d5eac2dd1d0a/package.json#L30

And since the author of colors.js is seemingly having a mental breakdown or something, this now happens to packages depending on the latest version of colors.js
https://github.com/Marak/colors.js/issues/285
https://github.com/Marak/colors.js/commit/074a0f8ed0c31c35d13d28632bd8a049ff136fb6
https://github.com/Marak/colors.js/commit/5d2d242f656103ac38086d6b26433a09f1c38c75

**Comment by DanielRuf** [comment]
You can also use resolutions:

https://github.com/Marak/colors.js/issues/285#issuecomment-1008168237

**Comment by karellm** [comment]
I've published `5.4.0` that fixes the issue by locking the dependency on `1.4.0`. It might be an invitation to lock all dependencies but I will wait for the dust to settle to make a decision on this.

**Comment by tanosugi** [comment]
Thanks for your quick response.
It was solved for me by update i18next-parser by following command because you fixed as above.
```js
npm uninstall -g i18next-parser
npm install -g i18next-parser
i18next
```
